src/components/ErrorModal.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="modal-mask"
+    @click="resolve"
+  >
+    <div class="modal-wrapper" @click.stop>
+      <div class="error-modal">
+        <h1>{{ $t('reportBug') }}</h1>
+        <textarea
+          v-model="report"
+          class="report-text"
+        />
+        <button
+          class="button"
+          @click="send"
+        >
+          {{ $t('sendBugReport') }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { detect } from 'detect-browser';
+
```

Comment:
```suggestion
```

Reply:
But how I will set https://github.com/aeternity/superhero-ui/blob/6a14ea86732f52fb0f947c3d62b0625128e39e9e/src/components/ErrorModal.vue#L49
the same lib used at the wallet